### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: src/FormModel.php [hunk 1]
```diff
@@ -262,26 +262,28 @@ private function addErrors(array $items): void
...
-
-                if ($type !== null) {
-                    $this->attributes[$property->getName()] = $type->getName();
-                } else {
-                    throw new \InvalidArgumentException(
-                        "You must specify the type hint for \"$property->class\" class."
-                    );
-                }
+            if ($property->isStatic()) {
+                continue;
```
Comment (terabytesoftw) [inline src/FormModel.php:272]: You are adding a mutant, just check that it is not static.

Comment (xepozz) [reply src/FormModel.php:272]: What?

Comment (samdark) [reply src/FormModel.php:272]: Likely it's about mutation testing and Infection.

Comment (terabytesoftw) [reply src/FormModel.php:272]: https://infection.github.io/guide/mutators.html#Loop

Comment (xepozz) [reply src/FormModel.php:272]: Foreach and for produces mutations too. We won't use it too?

Comment (xepozz) [reply src/FormModel.php:272]: image
I've fixed it.

**Comment by xepozz** [comment]
I've fixed some other mutants too

**Comment by samdark** [comment]
Thanks!

**Pull request merged**